Blazor tutorial UE pass (#20941)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/build-a-blazor-app.md

1. Open the `Todo` component in any file editor and add an `@page` Razor directive to the top of the file with a relative URL of `/todo`.

   `Pages/Todo.razor`:

   [!code-razor[](build-a-blazor-app/samples_snapshot/ToDo0.razor?highlight=1)]

   Save the `Pages/Todo.razor` file.

1. Add the `Todo` component to the navigation bar.

   The `NavMenu` component is used in the app's layout. Layouts are components that allow you to avoid duplication of content in an app. The `NavLink` component provides a cue in the app's UI when the component URL is loaded by the app.

   In the unordered list (`<ul>...</ul>`) of the `NavMenu` component, add the following list item (`<li>...</li>`) and `NavLink` component for the `Todo` component.

   In `Shared/NavMenu.razor`:

   [!code-razor[](build-a-blazor-app/samples_snapshot/NavMenu.razor?highlight=5-9)]

   Save the `Shared/NavMenu.razor` file.

1. Build and run the app by executing the [`dotnet watch run`](/aspnet/core/tutorials/dotnet-watch) command in the command shell from the `TodoList` folder. After the app is running, visit the new Todo page by selecting the **`Todo`** link in the app's navigation bar, which loads the page at `/todo`.

>    Leave the app running the command shell. Each time a file is saved, the app is automatically rebuilt. The browser temporarily loses its connection to the app while compiling and restarting. The page in the browser is automatically reloaded when the connection is re-established.

Reviewer: ```suggestion
    Leave the app running the command shell. Each time a file is saved, the app is automatically rebuilt and the application page is reloaded in the browser.
```

I think this can be simplified.
